aemedge/blocks/hero/hero.js

```diff
@@ -12,19 +18,37 @@ function calculateInitials(name) {
   return initials;
 }
 
-function decorateMetaInfo() {
-  const infoBlockWrapper = div({ class: 'media-blend__info-block' });
+function buildAuthorEl(author) {
+  return a({ class: 'media-blend__author', href: buildAuthorUrl(author) }, author.trim());
+}
 
-  const author = getMetadata('author');
-  if (author) {
-    const avatar = document.createElement('udex-avatar');
-    avatar.setAttribute('size', 'XS');
-    avatar.setAttribute('initials', calculateInitials(author));
-    avatar.setAttribute('color-scheme', 'Neutral');
+function decorateMetaInfo() {
+  const infoBlockWrapper = span({ class: 'media-blend__info-block' });
+
+  const authors = removeAuthorsSuffixes(getMetadata('author')).split(',');
```

Comment:
I tested https://387-hero-clickable-elements--hlx-test--urfuwo.hlx.page/draft/hupe/224-multiple-authors and we don't trim the author names. If I may suggest, I would add the following (see author-profile.js):

.map((e) => e.trim())

Reply:
does that make any difference? am I over-simplifying if I think that it's whitespace which is ignore in HTML?

Reply:
We are using the strings to build URLs, where the whitespace does make a difference. We should already be trimming the names in buildAuthorUrl and buildAuthorEl, but I can move that here for the sake of simplicity/readablity.

Reply:
We look up author entries in the index based on author names. We need precise mappings. Will be required e.g. for #292